gph/python/test/test_ripser.py

```diff
@@ -235,7 +235,7 @@ def test_gens_edge_in_dm_and_sorted():
 def test_gens_with_collapser():
     """This test ensures that you cannot use collapser and
     retrieve representative simplices. This is a temporary behavior."""
-    X = squareform(pdist(np.random.random((100, 3))))
+    X = squareform(pdist(np.random.random((10, 3))))
```

Comment:
I did this small update because we don't care about the input data, I don't want that the test are long just because we loose time in generating the input data when we are not interested in the results of the computation themself.

Reply:
Good catch, thanks!